projboard/views.py

```diff
@@ -57,3 +60,20 @@ def home_page(request):
 
 def about_page(request):
     return render(request, 'about/about.html', {})
+
+
+def login_view(request):
+    if request.method == 'POST':
+        form = LoginForm(request.POST)
+        if form.is_valid():
+            username = form.cleaned_data['username']
+            password = form.cleaned_data['password']
+            user = authenticate(request, username=username, password=password)
+            if user is not None:
+                login(request, user)
+                return redirect('/')
+            else:
+                messages.info(request, 'Invalid username or password')
+    else:
+        form = LoginForm()
+    return render(request, 'login/login.html')
```

Comment:
shouldn't you send the form to the template ??

Reply:
It didn't work for me when I did {'form': form}